t/ResultSet/Explain.t

```diff
@@ -30,4 +32,5 @@ run_me(SQLite => {
 run_me(Pg     => { engine => 'Pg'     });
 run_me(mysql  => { engine => 'mysql'  });
 
+ok $ran, 'tests were acutally run';
```

Comment:
I'd be a bit more explicit here. Something like: `'tests ran at least once against default RDBMS'`

Reply:
yeah, i'll get on that tomorrow.